Alicerce/Sources/Logging/Chronicle/Log.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  Alicerce.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 03/04/17.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+final class Log {
+    
+    fileprivate static var providers = Array<LogProvider>()
+
+    static let defaultLevel: Level = Level.error
+    
+    // MARK:- Provider Management
+    
+    static var providerCount: Int {
+        return providers.count
+    }
+    
+    public class func registerProvider(provider: LogProvider) {
```

Comment:
Following Swift 3 naming guidelines, this could be renamed to `register(_ provider: LogProvider)`.

From the point of calling, I already know that:
1. I'm registering something
2. That something is a `LogProvider`

What do you say?